src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
> There's so much I don't know about how to build FFmpeg that I don't try to guess what is actually supported or not

https://github.com/jb-alvarado/media-autobuild_suite ?

> hevc_cuvid was actually recognized 👍, but it doesn't support 10-bit

[Of course](https://trac.ffmpeg.org/wiki/HWAccelIntro#CUDACUVIDNvDecode) that will depend on your card cabilities. 

> so i suspect that perhaps you need to ensure Bios or UEFI settings force it to present both to the OS

I'm not sure even if we were talking about the ASIC module embedded in a CPU that'd be a toggleable setting